continuous_integration/recipes/distributed/meta.yaml

```diff
@@ -40,8 +40,8 @@ outputs:
       noarch: python                                                                          # [build_ext == "python"]
       script: >
         python -m pip install . -vv
-          --no-deps
-          --install-option="--with-cython=profile"  # [build_ext == "cython"]
+        --no-deps
+        --install-option="--with-cython=profile"  # [build_ext == "cython"]
```

Comment:
Was just about to suggest this 😄

Reply:
Followed up with a small consolidation of this in https://github.com/dask/distributed/pull/5831/commits/7866647fc7f28267142c2d09a3626ceb896e5e30, but it looks like this got builds passing 🎉